server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2436,6 +2436,9 @@ public HostVO fillRoutingHostVO(final HostVO host, final StartupRoutingCommand s
         host.setHypervisorType(hyType);
         host.setHypervisorVersion(ssCmd.getHypervisorVersion());
         host.setGpuGroups(ssCmd.getGpuGroupDetails());
+
+        s_logger.info(String.format("Filling %s with the following information retrieved from the agent {\"cpus\": %s, \"cpuSpeed\": %s, \"cpuSockets\": %s, "
```

Comment:
Logging may not be necessary if it's a command that is sent from agent. There is log statement that should cover it already (on either mgmt  server/agent side)

Reply:
You are right. 

Repeated logs were removed, thanks!